src/workers/parser/utils/ast.js

```diff
@@ -74,7 +78,9 @@ export function getAst(source: Source) {
   if (contentType == "text/html") {
     ast = parseScriptTags(source.text, htmlParser) || {};
   } else if (contentType && contentType.match(/(javascript|jsx)/)) {
-    ast = parse(source.text);
+    const type = source.id.includes("original") ? "original" : "generated";
```

Comment:
This is really smart! it doesn't make sense to use all the plugins on bundles and things that have been processed. 🚀